scripts/spot-grid.py

```
@@ -90,9 +90,10 @@ def new_image(self, data):
             self.app.rotation = tform_xy.rotation
             self.app.squeeze = tform_xy.squeeze
             self.app.shear = tform_xy.shear
-        except ValueError as err:
+        except (ValueError, IndexError) as err:
             logging.warning("No grid found on image, cannot display spots: %s", err)
-            self.app.spots = None
+        except Exception as err:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)